static/app/components/smartSearchBar/index.tsx

```diff
@@ -132,134 +132,144 @@ export type ActionBarItem = {
   makeAction: (props: ActionProps) => {Button: VFC; menuItem: MenuItemProps};
 };
 
-type Props = WithRouterProps & {
-  api: Client;
-  organization: Organization;
-  /**
-   * Additional components to render as actions on the right of the search bar
-   */
-  actionBarItems?: ActionBarItem[];
-  className?: string;
-
-  /**
-   * Custom Performance Metrics for query string unit parsing
-   */
-  customPerformanceMetrics?: CustomMeasurementCollection;
-  defaultQuery?: string;
+type DefaultProps = {
```

Comment:
i hoisted some fields up, and added `fieldDefinitionGetter`

'hide whitespace' in github is helpful here.